Documentation/network/kubernetes/bandwidth-manager.rst

```diff
@@ -292,6 +276,26 @@ probing you might observe a higher rate of TCP retransmissions compared to CUBIC
 We recommend to use BBR in particular for clusters where Pods are exposed as Services
 which serve external clients connecting from the Internet.
 
+BBR for The Host
+################
+In legacy routing mode, it is not possible to enable BBR for non-host-network
+pods (``hostNetwork: false``) for the reasons mentioned above; however, it is
```

Comment:
nit: non-host-network is a bit of a mouthful. We have a concept for this in Cilium documentation: Cilium-managed pods. For clarity we can keep the brackets.

Also whitespace following title.

```suggestion

In legacy routing mode, it is not possible to enable BBR for managed pods
(``hostNetwork: false``) for the reasons mentioned above; however, it is
```

Reply:
Done